roles/ee_builder/tasks/00_build_ee.yml

```
@@ -0,0 +1,90 @@
+---
+# tasks file for ee_builder
+- name: Build directory creation block.
+  block:
+    - name: Create temporary folder
+      ansible.builtin.tempfile:
+        state: directory
+        suffix: temp
+      register: build_dir
+
+    - name: Set Build dir fact.
+      ansible.builtin.set_fact:
+        builder_dir: "{{ build_dir.path }}"
+
+  when: builder_dir is not defined
+
+- name: Pull builder images
+  containers.podman.podman_image:
+    name: "{{ item }}"
+    username: "{{ base_registery_username | default(omit, true) }}"
+    password: "{{ base_registery_password | default(omit, true) }}"
+    validate_certs: "{{ ee_validate_certs | default(omit) }}"
+    force: true
+  loop:
+    - "{{ ee_base_image }}"
+    - "{{ ee_builder_image }}"
+  when: ee_update_base_images
+
+- name: ensure bindep file is rendered
+  ansible.builtin.template:
+    src: bindep.j2
+    dest: "{{ builder_dir }}/bindep"
+    mode: '0644'
+  when: __execution_environment_defintion.bindep is defined
+
+- name: Ensure python file is rendered
+  ansible.builtin.template:
+    src: python.j2
+    dest: "{{ builder_dir }}/python.txt"
+    mode: '0644'
+  when: __execution_environment_defintion.python is defined
+
+- name: Ensure requiremets file is rendered
+  ansible.builtin.template:
+    src: requiremets.yaml.j2
+    dest: "{{ builder_dir }}/requiremets.yaml"
```

If you put all these suggestions in then that would allow overriding if we set the default. (You also had a typo here too)
```suggestion
    dest: "{{ builder_dir }}/{{ galaxy_requirements_file }}"
```

---

Fixed